pkg/infra/remotecache/database_storage.go

```diff
@@ -102,15 +102,21 @@ func (dc *databaseCache) Set(key string, value interface{}, expire time.Duration
 			expiresInSeconds = int64(expire) / int64(time.Second)
 		}
 
-		// insert or update depending on if item already exist
-		if has {
-			sql := `UPDATE cache_data SET data=?, created_at=?, expires=? WHERE cache_key=?`
-			_, err = session.Exec(sql, data, getTime().Unix(), expiresInSeconds, key)
-		} else {
-			sql := `INSERT INTO cache_data (cache_key,data,created_at,expires) VALUES(?,?,?,?)`
+		sec := dc.SQLStore.Cfg.Raw.Section("database")
```

Comment:
using dc.SQLStore.Dialect.DriverName() might be better, or add DriverName function on SqlStore, there are constants as well for the driver names defined in sqlstore/types.go